docs/src/api/class-page.md

```diff
@@ -1479,6 +1479,18 @@ The page's main frame. Page is guaranteed to have a main frame which persists du
 
 Returns the opener for popup pages and `null` for others. If the opener has been closed already the returns `null`.
 
+## async method: Page.pause
+
+Pauses script execution. Playwright will stop the script and wait for the user to either press 'Resume' button in the
```

Comment:
This sounds strange in the async world. We are not "stopping the script" - rather, the method waits for something, like a `waitForTimeout`.

Perhaps `this method will wait until user presses "Resume" button in the page overlay or calls playwright.resume() in the DevTools console` is more accurate?

Reply:
I'll rephrase